src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Ui;
+
+use Sylius\Behat\Context\FeatureContext;
+use Sylius\Component\Core\Model\ShippingMethod;
+use Sylius\Component\Product\Model\ProductInterface;
+use Sylius\Component\Shipping\Calculator\DefaultCalculators;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class CheckoutContext extends FeatureContext
+{
+    /**
+     * @Given I added product :name to the cart
+     */
+    public function iAddedProductToTheCart($name)
+    {
+        /** @var ProductInterface $product */
+        $product = $this->getObjectWithSpecificValue('sylius.repository.product', array('name' => $name));
+
+        $productShowPage = $this->getPage('Product\ProductShowPage')->openSpecificProductPage($product);
+        $productShowPage->addToCart();
+    }
+
+    /**
+     * @When I proceed selecting :paymentMethodName payment method
+     */
+    public function iProceedSelectingOfflinePaymentMethod($paymentMethodName)
+    {
+        $zone = $this->clipboard->getCurrentObject('zone');
+        $country = $this->clipboard->getCurrentObject('country');
+        $zoneMember = $this->clipboard->getCurrentObject('zone_member_country');
+
+        /** @var ShippingMethod $shippingMethod */
+        $shippingMethod = $this->getService('sylius.factory.shipping_method')->createNew();
+        $shippingMethod->setEnabled(true);
+        $shippingMethod->setCode('SM1');
+        $shippingMethod->setName('DHL');
+        $shippingMethod->setCurrentLocale('US');
+        $shippingMethod->setConfiguration(array('amount' => 200));
+        $shippingMethod->setCalculator(DefaultCalculators::PER_ITEM_RATE);
```

Comment:
If step does not specify how much shipping method charges, we should not assume anything here. So it should be free shipping and that's it. So flat rate + 0.